runtime/runtime/src/balance_checker.rs

```diff
@@ -98,6 +100,7 @@ fn total_accounts_balance(
 enum PostponedReceiptType {
     Action,
     PromiseYield,
+    GlobalContractDistribution,
```

Comment:
global contract distribution receipt cannot be postponed since it doesn't have any dependencies